### ISSUE TITLE
Move ocs4 module up to be below Infrastructure Nodes and Operators module

### ISSUE THREAD

**Issue by netzzer** [openshift/openshift-cns-testdrive#485]
Move module Deploying and Managing OpenShift Container Storage up in the order of modules to be  be below Infrastructure Nodes and Operators module in the Homeroom labguide. New order shown below.

$ cat workshop.yml
```
name: OpenShift and Container Storage for Admins

modules:
  activate:
  - environment
  - installation
  - app-mgmt-basics
  - app-storage-basics
  - machinesets
  - infra-nodes
  - ocs4
  - logging
  - ldap-groupsync
  - monitoring-basics
  - template-quota-limits
  - networking
  - disabling-project-self-provisioning
  - clusterresourcequota
```

**Comment by thoraxe** [comment]
@netzzer didn't you do this with one of your PRs already?

**Comment by netzzer** [comment]
Fixed with #492
